### PR TITLE
AdminDashboard & Alerts update

### PR DESCRIPTION
**Pull request by Shawn2776**
Updated AdminDashboard Landing graph, and updated the Alerts page. Commented out the pagination bug, to be resolved later.

### CODE REVIEW

File: src/components/pages/AdminDashboard/AdminDashboardPage.js [hunk 1]
```diff
@@ -98,9 +98,7 @@ function RenderHomePage() {
 
   function graph() {
     axios
-      .get(
-        'http://bg-ds-api-dev.us-east-1.elasticbeanstalk.com/vis/pie/Anderson/Basketball/2021-07-01/2021-07-31'
-      )
+      .get('http://bg-ds-api-dev.us-east-1.elasticbeanstalk.com/vis/daily/')
       .then(res => {
         console.log('response res.data: ', JSON.parse(res.data));
```
Comment (fidan93) [inline src/components/pages/AdminDashboard/AdminDashboardPage.js:103]: If this function is already working we can remove console.log.

**Review by fidan93 — APPROVED** [review]
All pagination code was commented out.  Is that supposed to be that way?

File: src/components/pages/AdminDashboard/AdminDashboardPage.js (hunk 1 above)
Comment (Shawn2776) [inline src/components/pages/AdminDashboard/AdminDashboardPage.js:103]:
```suggestion
        
```

Comment (Shawn2776) [reply src/components/pages/AdminDashboard/AdminDashboardPage.js]: Resolved below. Thank You

**Pull request merged**